docs/guide/controller-start-up.md

```diff
@@ -2,10 +2,10 @@
 
 The .wbt file contains the name of the controller that needs to be started for each robot.
 The controller name is a platform and language independent field; for example when a controller name is specified as "xyz\_controller" in the .wbt file, this does not say anything about the controller's programming language or platform.
-This is done deliberately to ensure the platform and programming language independence of .wbt files.
+This is done deliberately to ensure the independence between platform and programming language of .wbt files.
```

Comment:
Are you sure that this change is correct?
Because in the new sentence to me it seems that platform and programming languages should be independent.
But in fact are are WBT files that are independent from the platform and they are independent from the programming language used.

Reply:
That sounds wrong to me as well...

Reply:
@PRUBHTEJ: can you propose a rephrasing for this?

Reply:
@omichel , @stefaniapedrazzi  I'll try to rephrase it as soon as possible .